.versions

```diff
@@ -9,8 +9,8 @@ GO_DEP_VERSION=0.5.4
 GO_VERSION=1.21.1
 HASKELL_STACK_VERSION=2.7.5
 JAVA_VERSION=17
-NODEJS_VERSION=18.17.1
-NPM_VERSION=8.15.1
+NODEJS_VERSION=20.9.0
```

Comment:
Commit message nit:

> Use the LTS version

Maybe say "*current* LTS version" as the LTS version changes over time.